File: MeTalk/Recorder/Cells/MTAudioTableViewCell.m
//
//  MTAudioTableViewCell.m
//  MeTalk
//
//  Created by yu chung hyun on 2016. 3. 7..
//  Copyright © 2016년 yu chung hyun. All rights reserved.
//

#import "MTAudioTableViewCell.h"
#import "UCPlotView.h"
#import "MTMacros.h"

@interface MTAudioTableViewCell()


@property (weak, nonatomic) IBOutlet UILabel *lblTime;
@property (weak, nonatomic) IBOutlet UCPlotView *plotView;

@end

@implementation MTAudioTableViewCell

- (void)awakeFromNib {
    // Initialization code
//UIEdgeInsetsMake(20, 5, 5, 10);
//UIEdgeInsetsMake(20, 10, 5, 5)
    
    self.contentView.backgroundColor = ColorFromRGBA(0xf0, 0x00, 0x00, 0.1);
    self.plotView.backgroundColor = [UIColor clearColor];
    self.plotView.plotColor = ColorFromRGBA(0x9c,0x00,0x00,0.9);
    self.plotView.plotWidth = 2.0f;
    self.plotView.mode = UCPlotViewModeProgress;
}

- (void)setSelected:(BOOL)selected animated:(BOOL)animated {
    [super setSelected:selected animated:animated];

    // Configure the view for the selected state
}

- (void)setTime:(NSString *)str
{
    [self.lblTime setText:str];
}

- (void)setPlots:(NSArray *)plots
{
    [self.plotView setPeeks:plots];
}
@end

File: MeTalk/Recorder/Cells/MTAudioTableViewCell.swift
//
//  MTAudioTableViewCell.swift
//  MeTalk
//
//  Created by yu chung hyun on 2016. 5. 11..
//  Copyright © 2016년 yu chung hyun. All rights reserved.
//

import UIKit

@IBDesignable
class MTAudioTableViewCell: UITableViewCell {

    @IBOutlet weak var plotView: UCPlotView!
    @IBOutlet weak var lblTime: UILabel!
    
    override func awakeFromNib() {
        super.awakeFromNib()
        // Initialization code

        
        
        
        self.contentView.backgroundColor = UIColor.colorRGB(0xf0, 0x00, 0x00,0.1)
        self.plotView.backgroundColor = UIColor.clearColor()
        self.plotView.plotColor = UIColor.colorRGB(0x9c, 0x00, 0x00,0.9)
        
        self.plotView.plotWidth = 2.0
        self.plotView.mode = UCPlotViewModeProgress
        
    }

    override func setSelected(selected: Bool, animated: Bool) {
        super.setSelected(selected, animated: animated)

        // Configure the view for the selected state
    }
  
    func setTime(str:String) -> Void {
        self.lblTime.text = str
    }
    
    func setPlots(plots:Array<AnyObject>?) -> Void {
        self.plotView.setPeeks(plots)
    }

}
